sms/src/main/java/org/motechproject/sms/http/MultilineResponseHandler.java

```diff
@@ -77,7 +77,7 @@ public void handle(OutgoingSms sms, String response, Header[] headers) {
                 getLogger().info(String.format("Sent messageId %s '%s' to %s", messageId, messageForLog(sms),
                         recipient));
                 getAuditRecords().add(new SmsRecord(getConfig().getName(), OUTBOUND, recipient, sms.getMessage(), now(),
-                        DeliveryStatus.DISPATCHED, null, sms.getMotechId(), messageId, null));
+                        getTemplateOutgoingResponse().getSuccessStatus(), null, sms.getMotechId(), messageId, null));
```

Comment:
According to the javadocs, this field contains the status, that the provider considers as success. Are you sure this is the proper field to use? Won't this always put the success in the logs, no matter what the actual response was?